project/up5k-nes.json

```diff
@@ -0,0 +1,34 @@
+{
+    "srcs": [
+	"src/up5k-nes/MicroCode.v",
```

Comment:
I think this should end up in the third_party directory as well. Given that up5k demo is a submodule, you may want to create a third_party dir like `up5k-demos-changes`, where you put the MicroCode design with the fixes for the initial `_` values.

This should be similar to [ibex](https://github.com/SymbiFlow/fpga-tool-perf/tree/master/third_party/ibex) for instance, where there are, the modified files, a `README.fpga-tool-perf` file indicating the changes made, and the up5k-demo LICENSE file.

Reply:
Ok, I moved this file to third party dir and added both ``README.fpga-tool-perf`` and ``LICENSE`` files.